### PR TITLE
Updated examples for create tickets (including updating to advanced-d…

### PR DESCRIPTION
**Pull request by ANlKA**
Updated examples for ticket creation with correct variable values (Eg: I saw --diagnostic-consent in some examples) 
Tested quota examples
Deleted quota examples from no subscription
Run updated code style check and fix errors (Passed all of Pylint but not Flake8 since the issues were minor and hard to identify or fix)
Reran live tests for support under Prod env and added that to the PR 
Hid quota related params in No-Sub

### CODE REVIEW

File: src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py [hunk 1]
```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
...
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/sql_database_problemClassification_guid" --severity "moderate" --ticket-name "SQLDatabaseDTUsQuotaIncreaseRequest" --title "Request for DTUs Quota Increase in SQL Database" --quota-change-version "1.0" --quota-change-request-sub-type "DTUs" --quota-change-requests '[{"region":"EastUS","payload":"{"ServerName":"testserver","NewLimit":54000}"}]'
```
Comment (RudraSharma93Microsoft) [inline src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py]: --quota-change-requests '[{"region":"EastUS","payload":"{"ServerName":"testserver","NewLimit":54000}"}]' this is not what we had agreed upon, please update to the expected value

File: src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py [hunk 2]
```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
...
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "Foo" --contact-language "en-us" --contact-last-name "Bar" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "QuotaTicketDescription" --advanced-diagnostic-consent  "Yes" --problem-classification  "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/batch_problemClassification_guid" --severity "minimal" --ticket-name "BatchPoolsQuotaIncreaseRequest" --title "Request for Pools Quota Increase in Batch Account" --quota-change-version "1.0" --quota-change-requests [0].region="WESTUS" --quota-change-requests [0].payload="{AccountName: test, NewLimit: 102, Type: Pools}" --quota-change-subtype "Account"
```
Comment (RudraSharma93Microsoft) [inline src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py:57]: "{AccountName: test, NewLimit: 102, Type: Pools}" is space okay? Yesterday we had noticed it was complaing about space between characters

File: src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py [hunk 3]
```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
...
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/batch_problemClassification_guid" --severity "moderate" --ticket-name "MachineLearningVMFamilyCoresQuotaIncreaseRequest" --title "Request for VM Family Cores Quota Increase in Machine Learning Service" --quota-change-version "1.0" --quota-change-subtype "BatchAml" --quota-change-requests "[{region:'EastUS',payload:'{VMFamily:standardA0_A7Family,NewLimit:200,Type:Dedicated}'}]"
```
Comment (RudraSharma93Microsoft) [inline src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py]: Should this guid be machine_learning_service_problemClassification_guid

File: src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py [hunk 4]
```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
...
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "abc" --contact-language "en-us" --contact-last-name "xyz" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "my description" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/machine_learning_service_problemClassification_guid" --severity "moderate" --ticket-name "BatchVMFamilyCoresQuotaIncreaseRequest" --title "Request for VM Family Cores Quota Increase in Batch Account" --quota-change-version "1.0" --quota-change-subtype "Account" --quota-change-requests "[{region:'EastUS',payload:'{AccountName:test,VMFamily:standardA0_A7Family,NewLimit:200,Type:Dedicated}'}]"
```
Comment (RudraSharma93Microsoft) [inline src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py]: Shouldnt this guid be batch_problemClassification_guid

File: src/support/azext_support/custom.py [hunk 5]
```diff
@@ -326,8 +439,8 @@ def pre_operations(self):
 def upload_files_no_subscription(cmd, file_path, file_workspace_name):
 
     from .aaz.latest.support.no_subscription.file import (
-        Create as Create,
-        Upload as Upload,
+        Create,
```
Comment (RudraSharma93Microsoft) [inline src/support/azext_support/custom.py:442]: Does this change work?

Comment (ANlKA) [reply src/support/azext_support/custom.py:442]: Yes, we just aren't aliasing anymore

File: src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py (hunk 2 above)
Comment (ANlKA) [reply src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py:57]: image
No it's fine

**Review by RudraSharma93Microsoft — APPROVED** [review]

**Pull request merged**